tasks/src/zero-shot-image-classification/data.ts

```diff
@@ -0,0 +1,48 @@
+import type { TaskDataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			// TODO write proper description
+			description: "",
+			id: "",
+		},
+	],
+	demo: {
+		inputs: [],
+		outputs: [],
+	},
+	metrics: [
+		{
+			description:
+				"Computes the number of times the correct label appears in top K labels predicted",
+			id: "top-K accuracy",
+		},
+	],
+	models: [
+		{
+			// TO DO: write description
+			description:
+				"Robust image classification model trained on publicly available image-caption data.",
+			id: "openai/clip-vit-large-patch14",
+		},
+		{
+			// TO DO: write description
+			description:
+				"Robust image classification model trained on publicly available image-caption data.",
+			id: "openai/clip-vit-base-patch16",
+		},
+		{
+			// TO DO: write description
+			description:
+				"Robust image classification model trained on  LAION-2B English subset of LAION-5B (https://laion.ai/blog/laion-5b/) using OpenCLIP (https://github.com/mlfoundations/open_clip).",
```

Comment:
I would avoid putting this model here given it's known to have sexual biases 😅

Reply:
Oops didn't know that...I removed the model now